sample_factory/algorithms/appo/appo.py

```diff
@@ -127,42 +127,43 @@ def arg(*args, **kwargs):
-                            'no difference between an action distribution with a probability epsilon > 0 for '
-                            'some action and an action distribution with a probability = zero for this action.'
-                            ' For many tasks the first (epsilon) distribution is preferrable because we keep some '
-                            '(albeit small) amount of exploration, while the second distribution will never explore '
-                            'this action ever again.'
-                            'Unlike the entropy term, symmetric KL divergence between the action distribution '
-                            'and a uniform prior approaches infinity when entropy of the distribution approaches zero,'
-                            ' so it can prevent the pathological situations where the agent stops exploring. '
-                            'Empirically, symmetric KL-divergence yielded slightly better results on some problems.',
-                       )
-
-        # Max entropy PPO
-        arg('--ppo_max_entropy_coeff', default=0.1, type=float, 
-                        help='Coefficient for max entropy term added to PPO.0 means no max entropy term to env rewards.'
-                             'Note that this is different from exploration loss.'
-                        )
+            help='Usually the exploration loss is based on maximizing the entropy of the probability'
+                 ' distribution. Note that mathematically maximizing entropy of the categorical probability '
+                 'distribution is exactly the same as minimizing the (regular) KL-divergence between'
+                 ' this distribution and a uniform prior. The downside of using the entropy term '
+                 '(or regular asymmetric KL-divergence) is the fact that penalty does not increase as '
+                 'probabilities of some actions approach zero. I.e. numerically, there is almost '
+                 'no difference between an action distribution with a probability epsilon > 0 for '
+                 'some action and an action distribution with a probability = zero for this action.'
+                 ' For many tasks the first (epsilon) distribution is preferrable because we keep some '
+                 '(albeit small) amount of exploration, while the second distribution will never explore '
+                 'this action ever again.'
+                 'Unlike the entropy term, symmetric KL divergence between the action distribution '
+                 'and a uniform prior approaches infinity when entropy of the distribution approaches zero,'
+                 ' so it can prevent the pathological situations where the agent stops exploring. '
+                 'Empirically, symmetric KL-divergence yielded slightly better results on some problems.',
+        )
+
+        # Max entropy objective
+        arg(
+            '--max_entropy_coeff', default=0.0, type=float,
```

Comment:
this renaming is fine, depends on what all you plan to include for SF2. This PR was only changing things for APPO

Reply:
Old name was fine too, but the parameter itself is not PPO-specific, so shorter name should be fine.